service/endpoint.cc

```diff
@@ -64,6 +64,16 @@ EndpointBase::
 
 void
 EndpointBase::
+setPollingMode(enum PollingMode mode)
```

Comment:
I know that, but I meant writing `enum PollingMode mode` rather than `PollingMode mode`